src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -276,7 +276,7 @@ function PolicyAccountingPage({policy}: PolicyAccountingPageProps) {
                     : {
                           description: translate('workspace.intacct.entity'),
                           iconRight: Expensicons.ArrowRight,
-                          title: getCurrentSageIntacctEntityName(policy),
+                          title: getCurrentSageIntacctEntityName(policy) ?? translate('workspace.common.topLevel'),
```

Comment:
We should modify `getCurrentSageIntacctEntityName` function to return `top level` as a default